### DISCUSSION TITLE
Kiai section in time bar do not refeash when set second kiai effect

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#19404] [Q&A] [answered]
kiai section have two timing line, one of its start and another of its end
when set a timing line to end kiai sction, time bar don't refresh like 
![image](https://user-images.githubusercontent.com/34775378/180997490-38e9947d-5acf-46ee-82c4-a036ee83b6c1.png)
I think it's a problem with the refresh timing of the time bar, it should refresh all kiai section after the second line(or any) is placed

https://user-images.githubusercontent.com/34775378/180997912-a12053f7-4254-4ee7-bb8b-6d33ba3d5c32.mp4

version: 2022.723.0

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/19412, will be resolved by https://github.com/ppy/osu/pull/19457
